app/scripts/lib/oauth-client.js

```diff
@@ -44,8 +45,37 @@ define(function (require, exports, module) {
     getClientInfo: function getClientInfo(id) {
       return this._request('get', GET_CLIENT + id);
     },
+    /**
+     * Fetch user's active OAuth clients
+     *
+     * @param {String} accessToken
+     * @returns {Promise}
+     */
+    fetchClients: function fetchClients(accessToken) {
```

Comment:
You can use object shorthand here and babel will add the function name for you, also space between `fetchClients` and `(`.

```js
fetchClients (accessToken) {
...
}
```